main.py

```diff
@@ -0,0 +1,22 @@
+"""
+Week 8
+This is to test how many hours a grocery store is open a week
+"""
+
+class Grocery_Store:
+    """
+    bla
+    """
+    def __init__(self, hours, days):
+        self.hours = hours
+        self.days = days
+
+def total_hours(self):
+    """
+    This calculates weekly total hours of operation
+    """
+    weekly_hours = self.hours * self.days
+    return weekly_hours
+
+if __name__ == "__main__":
+    ten_hours = Grocery_Store(10, 7)
```

Comment:
## Grading

| Points | Location        | Criteria                                    |
| ------ | --------------- | ------------------------------------------- |
| 5     | main.py         | Protected variable set when class initiated |
| 8     | main.py         | Function that uses protected variable       |
| 10     | pytest coverage | Tests cover all aspects of main.py          |
| 8     | pylint          | Pylint score >= 8.0.                        |
|  -1      | instructor      | Code style, efficiency, and readability     |
| 30/40 |

Fixing the indenting will most likely resolve a lot of the issues that you are seeing/encountering and make this easier to do for Labs 9/10